src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+@use "../utils/variables" as *;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  border: 1px solid $gray-element;
+  border-radius: 5px;
+  background-image: url("../images/imac.jpeg");
+  background-position: center 32px;
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  padding: 16px;
+  width: 200px;
+  height: 406;
+  min-height: 100px;
+  color: $main-accent;
+  font-weight: 400;
+  font-size: 10px;
+
+  &__name {
+    margin-top: 190px;
+    width: 166px;
```

Comment:
you shouldn't define such pixels, just spaces between element, it also would be easier to do when you add `<img>` to your card